app/test_main.py

```diff
@@ -1,3 +1,19 @@
 from app.main import get_human_age
 
-# write your code here
+
+import pytest
+from app.main import get_human_age
```

Comment:
The `get_human_age` function is imported twice. You can remove one of these import statements to avoid redundancy.